### ISSUE TITLE
[Vue warn]: Error in render function: "TypeError: Cannot read property 'aria-rowcount' of undefined"

### ISSUE THREAD

**Issue by stonkeep** [bootstrap-vue/bootstrap-vue#1801]
I'm with the following error
`Error in render function: "TypeError: Cannot read property 'aria-rowcount' of undefined"`

My template 
```
 <b-table show-empty
                         aria-rowcount="5"
                         :items="items"
                         :fields="fields"
                         :current-page="currentPage"
                         :per-page="perPage"
                ></b-table>
```

and my script
```
<script>
  const items = [
    { isActive: true, age: 40, first_name: 'Dickerson', last_name: 'Macdonald' },
    { isActive: false, age: 21, first_name: 'Larsen', last_name: 'Shaw' },
    { isActive: false, age: 89, first_name: 'Geneva', last_name: 'Wilson' },
    { isActive: true, age: 38, first_name: 'Jami', last_name: 'Carney' }
  ]

  export default {
    data () {
      return {
        items: items
      }
    }
  }
</script>
```

I don't know what to do

**Comment by pdagosto** [comment]
I'm seeing this also. The problem seems to be that this.$attrs is undefined when the aria-rowcount property is accessed during initial display of the table:

    // Assemble table
    var table = h('table', {
      class: this.tableClasses,
      attrs: {
        id: this.safeId(),
        role: this.isStacked ? 'table' : null,
        'aria-busy': this.computedBusy ? 'true' : 'false',
        'aria-colcount': String(fields.length),
        **'aria-rowcount': this.$attrs['aria-rowcount'] || this.items.length > this.perPage ? this.items.length :** null
      }
    }, [caption, colgroup, thead, tfoot, tbody]);

I don't see this with the examples only with my application and I can't seem to reproduce it outside of my application. Looking into the Vue debug tools the fields and data are being received by the b-table component so that's not the problem. I tried tracing it through the debugger but can't figure out why this is not hit in the examples but is in my application. Unfortunately I haven't been able to build a smaller example outside of my application that reproduces the behavior.
